test/integration/notation/cases.yaml

```diff
@@ -0,0 +1,11 @@
+# simple test cases for notation
```

Comment:
we should add additional test cases for required timestamp failing due to no timestamp verification

maybe the other verification levels may be interesting, too, but the ts thing is most relevant as it ensures that the highest security setting works